pkg/apis/kops/componentconfig.go

```diff
@@ -581,6 +581,10 @@ type KubeControllerManagerConfig struct {
 	ConcurrentServiceaccountTokenSyncs *int32 `json:"concurrentServiceaccountTokenSyncs,omitempty" flag:"concurrent-serviceaccount-token-syncs"`
 	// The number of replicationcontroller objects that are allowed to sync concurrently.
 	ConcurrentRcSyncs *int32 `json:"concurrentRcSyncs,omitempty" flag:"concurrent-rc-syncs"`
+	// AuthenticationKubeconfig is the path to a Authentication Kubeconfig
```

Comment:
nit:

```suggestion
	// AuthenticationKubeconfig is the path to an Authentication Kubeconfig
```

and similar for below...

Reply:
fixed